### DISCUSSION TITLE
NFT的概況簡介

### DISCUSSION THREAD

**Discussion by hapiwangy** [1101-GS4719-C/Python-Community-GS4719-C#462] [General]
最近經常在各式各樣的地方砍到NFT這個詞的出現，那麼他究竟是甚麼呢?簡單來說，他是一種數位資產，基本上任何的資產都可以用NFT來做表示。NFT的實現方式需要透過區塊戀的這個技術，受限於篇幅大小，以後有機會再和大家分享。NFT的特點就是任何人都可以去做交易/發行/擁有。我認為在本文的例子當中，讓人可以擁有商品部分的擁有權進而可以進行不同的操作是令我感興趣的。另外，隨著此類技術的成熟和興起，有人把越來越多的東西進行NFT化，甚至可以把自己的時間進行交易，這是前所未聞!!以下這篇文章帶我初步了解了NFT的香冠應用，也希望將來有機會接觸到更多NFT技術層面的知識
[https://www.techbang.com/posts/90169-nft-art-token](https://www.techbang.com/posts/90169-nft-art-token)

**Comment by chupochun** [comment]
感謝分享NFT的基本介紹，讓我這個門外漢都能有基本的了解

**Comment by sean1325418** [comment]
看起來簡單說就是一種輕鬆交易數位資產的手段，它的好處就是可以很輕易就上手，任何人都可以使用

**Comment by xkllkx** [comment]
謝謝樓主的分享，讓我對NFT有了基礎的認識。

**Comment by bloodtrail0511** [comment]
感覺NFT就是虛擬世界中的所有權證書，只有你擁有了憑證，你手中的虛擬資產才是正品

**Comment by ghost** [comment]
最近IG好多畫家也都開設NFT以增加開源，創作者的福音

**Comment by yuhao112** [comment]
目前就我所觀察多炒個熱度，而非是真正了解此項技術的含意。因此才會仍舊有許多人把此技術當作是一個JPG檔。以我的了解真正NFT莫大的好處無非是對於交易所得到物品的真實性以及背後交流的透明度。但缺點也就是因為如此透明，加上每筆交易時都是要經過一整串繁雜的驗證過程，因此GAS fee以及時間確實是浪費了

**Comment by ThomasMeng0515** [comment]
NFT被吵得很火熱，之前都沒有深入去了解有關於NFT的內容，所以都不知道某些專業術語，謝謝分享這篇文章，讓我有機會深入認識NFT

**Comment by yucheng1028** [comment]
感謝版主讓人認識基礎的NFT知識

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我對NFT有初步的認識。

**Comment by d26488399** [comment]
感謝樓主分享，讓我對NFT有初步認識。

**Comment by HaoFish123** [comment]
謝謝分享，讓我對NFT有更進一步的認識。

**Comment by Sazmafuji** [comment]
NFT這最新的事業我認為只要metaverse有確切的發展，NFT的價值會暴漲，值得好好關切

**Comment by jason79461385** [comment]
前陣子有接觸過NFT相關的遊戲，但說實話不是很好玩就是了，感覺NFT現在都變成那些有錢人拿來炒的了。

**Comment by Rosancl** [comment]
感謝分享，讀完之後對NFT真的有比較全面的了解一點。

**Comment by Lai27330** [comment]
對於創作者來說能直接獲利還蠻棒的，但感覺上面的價格大多高於實際商品價值，有點像炒價的另類手段。

**Comment by yenwei12138** [comment]
到目前為止，多數NFT創作者也只是在炒熱度，趁這技術剛起步時大撈一筆

**Comment by treeann** [comment]
感謝樓主的分享 NFT是我關注很久的議題，主要擔心的是泡沫化的方面
